go/sgauth/credentials/file.go

```diff
@@ -90,8 +93,8 @@ func (f *File) TokenSource(ctx context.Context, scopes []string,
 			ClientSecret: f.ClientSecret,
 			Scopes:       scopes,
 			Endpoint:     internal.Endpoint{
-				AuthURL: f.AuthURL,
-				TokenURL: f.TokenURL,
```

Comment:
We should not use hardcoded urls here. How about using the default urls as fallback when the file url is empty?

Reply:
Thats totally makes sense. I assumed ADC file wont have any auth & token urls. Let me update the change accordingly.